Loops/Problem13/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Problem13
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            string binaryNum = Console.ReadLine();
+
+            double result = 0;
+
+            for (int i = 0; i < binaryNum.Length; i++)
+            {
+                int toThePowerOf = binaryNum.Length - i - 1;
```

Comment:
Leave comments to explain the behavior. What I will expect if %10 and /10 like in previous example or  binaryNum[i]) * Math.Pow(2, i)